src/utils/dotnetUtils.ts

```diff
@@ -26,7 +27,20 @@ export namespace dotnetUtils {
         return await getPropertyInProjFile(projFilePath, 'TargetFramework');
     }
 
-    export async function getPropertyInProjFile(projFilePath: string, prop: string): Promise<string> {
+    export async function getFuncVersion(projFilePath: string): Promise<FuncVersion | undefined> {
+        const version: string = await getPropertyInProjFile(projFilePath, 'AzureFunctionsVersion');
+        return tryParseFuncVersion(version);
+    }
+
+    export async function getPlatformTarget(projFilePath: string): Promise<string | undefined> {
```

Comment:
If a function returns `undefined` instead of throwing an error, it should have "try" at the beginning of the name

Reply:
Acknowledged